Livraria/src/Entity/Autores.java

```diff
@@ -0,0 +1,30 @@
+package Entity;
+
+import java.util.Date;
+
+public class Autores {
+	private String email;
+	private String nome;
+	private Date data;
+	
+	public Date getData() {
+		return data;
+	}
+	public void setData(Date data) {
+		this.data = data;
+	}
+	public String getEmail() {
+		return email;
+	}
+	public void setEmail(String email) {
+		this.email = email;
+	}
+	public String getNome() {
+		return nome;
+	}
+	public void setNome(String nome) {
+		this.nome = nome;
+	}
```

Comment:
Você precisa de get e set para tudo? Por que?

Reply:
Sim, preciso para manter o encapsulamento das variáveis, elas não podem ser públicas pois não podem ser acessadas de qualquer lugar se não iria furar o encapsulamento, então eu as declaro como privadas e preciso de um meio de acessá-las, então uso os get e setters.

Reply:
Encapsulamento não quer dizer só usar private com get e set, encapsulamento também é não ter get e set para tudo. get e set do jeito que está, não tem encapsulamento nenhum, justo por que eu posso acessar tudo e mudar tudo sem nenhuma restrição, o get ou o set não estão impedindo ou validando nada.

Reply:
Eu pensei que nesse caso o get e set só servem para o acesso, e tem tantos pois preciso acessar todas as variáveis da classe, e escolhi esse meio para acessar.

Reply:
Você precisa mesmo acessar todas as variáveis? Não vi você acessando todas elas no seu código.

Reply:
Eu acessei nome, email e o data a gente seta a data, só não pega ela, o que eu fiz foi deixar eles preparados, não sei se é errado deixar o get e set preparados porque sei que vou usar eles mais pra frente no projeto, se for, pode me explicar melhor?

Reply:
Na verdade você não sabe se vai usar o get ou o set mais na frente no projeto, você leu todos os cards? Então, quando você precisar, você escreve esse código. Atente-se à escrever o código que você precisa escrever, não o que você acha que vai precisar.